include/RMGHardware.hh

```diff
@@ -44,23 +46,70 @@ class RMGHardware : public G4VUserDetectorConstruction {
     RMGHardware(RMGHardware&&) = delete;
     RMGHardware& operator=(RMGHardware&&) = delete;
 
+    /** @brief Construct the detector
+     *
+     *  @details All detector geometry must be based on GDML files, parsed with @c G4GDMLParser .
```

Comment:
Not true. There always is the possibility to implement geometry in C++, as done in many of our examples.

also please state that this must not modify thread-local state, because it is only called once globally (I suppose?)